generator-test/src/test/java/com/linkedin/pegasus/generator/test/pdl/fixtures/CustomRecord.java

```diff
@@ -60,6 +60,11 @@ public int hashCode()
     return result;
   }
 
+  public static CustomRecord.Fields fields()
+  {
+    return new Fields();
```

Comment:
How come the fields has not something related with `title` and `body`?

Reply:
This is just a convenient API to get the nested "Fields" class. This is the current generated class for building projection masks.